vision_agent/tools/tools.py

```diff
@@ -26,6 +27,8 @@
     rle_decode,
 )
 
+register_heif_opener()
```

Comment:
it also needs to be executed inside of the sandbox.
so don't forget to copy this line to here: https://github.com/landing-ai/vision-agent/blob/main/vision_agent/agent/vision_agent.py#L44-L46

Reply:
oh good catch!